src/main/resources/standardTestCases/space/space.wdl

```diff
@@ -0,0 +1,48 @@
+task hello {
+    File f
+    String s = read_string(f)
+    command {
+        echo "Hello ${s}" > 'out with space.txt'
+    }
+    runtime {
+        docker: "ubuntu:latest"
+    }
+    output {
+        File single = "out with space.txt"
+        Array[File] globbed = glob("*.txt")
+    }
+}
+
+task goodbye {
+    File f
+    Array[File] files
+    command {
+        echo "Goodbye ${f}"
+        echo "Goodbye ${sep = " " files}"
+    }
+    runtime {
+            docker: "ubuntu:latest"
+    }
+    output {
+        String out = read_string(stdout())
+    }
+}
+
+workflow space {
+    File file_input = "gs://cloud-cromwell-dev/travis-centaur/file name with spaces.txt"
+    File file_input2 = "gs://cloud-cromwell-dev/travis-centaur/file%20name%20with%20.txt"
```

Comment:
I guess they could be created by a task. I just realized that to work around this exact problem we create all our files in tasks which doesn't provide much testing from input files coming from the input json or the declarations themselves.